shared/templates/sshd_lineinfile/tests/wrong_value_different_includings.fail.sh

```diff
@@ -2,8 +2,9 @@
 
 # platform = Oracle Linux 8,Oracle Linux 9
 
-SSHD_PARAM={{{ PARAMETER }}}
-SSHD_VAL="bad_val"
+{{% if XCCDF_VARIABLE %}}
```

Comment:
Would it make sense to extend all the Oracle test scenarios also to RHEL? How can we test that these scenarios work well and don't break Oracle?

Reply:
I don't think this is in scope of this PR. Oracle has an extra test in the oval which has something to do with the fact that there should be correct include directive present in /etc/ssh/sshd_config. It might be worth adding also to our OVAL, but I think this needs an extra PR.